telegram/_message.py

```diff
@@ -267,6 +269,8 @@ class Message(TelegramObject):
             or from anonymous administrators, information about the original sender chat.
         forward_from_message_id (:obj:`int`): Optional. For forwarded channel posts, identifier of
             the original message in the channel.
+        forward_sender_name (:obj:`str` :noindex: ): Optional. Sender's name for messages
+            forwarded from users who disallow adding a link to their account in forwarded messages.
```

Comment:
no need of re-documenting this as clot pointed out

Reply:
done in 
9fab68e49510ed8e623897941121f8742c80863e